plugins/pulp_rpm/plugins/migrations/0042_add_suse_fields_to_errata.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from pulp.server.db.connection import get_collection
+
+
+_logger = logging.getLogger('pulp_rpm.plugins.migrations.0042')
+
+
+def migrate(*args, **kwargs):
+
+    erratum_collection = get_collection('units_erratum')
+
+    for errata in erratum_collection.find({}):
+        erratum_collection.update({'_id': errata['_id']},
```

Comment:
For migration to be idempotent, I suggest to update only those which don't have new fields at all.